src/gan/config.cfg

```diff
@@ -1,38 +1,40 @@
 [GAN]
 ROLLOUT_UPDATE_RATE = 0.8
-ROLLOUT_NUM = 16
+ROLLOUT_NUM = 3
 
-G_STEPS = 1
+G_STEPS = 2
 D_STEPS = 5
-K_STEPS = 15
+K_STEPS = 3
 
 SEED = 88
 BATCH_SIZE = 64
-TOTAL_EPOCHS = 200 
-GENERATED_NUM = 100
+TOTAL_EPOCHS = 200
+GENERATED_NUM = 1000
 VOCAB_SIZE = 10
-SEQUENCE_LEN = 20
+SEQUENCE_LEN = 10
 
 data_dir = data
 glove_dir = ../../models
 glove_file = DA_glove_embeddings_300.pkl
```

Comment:
Just be careful -- this isn't likely to change, but if there is no way to pass this is as a run-time argument, then it's possible a lot of work you think is using a different glove file is not.